### DISCUSSION TITLE
Undelete account

### DISCUSSION THREAD

**Discussion by Bilis518** [pspete/psPAS#550] [Q&A]
Hi everyone,

I didn't find the way to undelete an account. (also no discussion found on this topis :-( 
Is that possible from psPas?

Thanks

**Comment by NathanielMaier** [comment]
This is a CyberArk limitation. There doesn't seem to be a way to do this in the new (v10) interface - only in the classic interface and PACLI. Therefore, psPAS (which implements the REST API) cannot do this. If you want to use PowerShell for this, check out pspete's PoShPACLI repo.
